lib/Common/Memory/HeapAllocator.h

```diff
@@ -130,6 +130,35 @@ struct HeapAllocator
     static HeapAllocator Instance;
     static HeapAllocator * GetNoMemProtectInstance();
 
+private:
+    // TODO (doilij) add config flags to enable/disable PrivateHeap functionality
+    HANDLE PrivateHeap;
+
+    void InitPrivateHeap()
+    {
+        if (this->PrivateHeap == nullptr)
+        {
+            PrivateHeap = HeapCreate(0, 0, 0); // no options, default initial size, no max size
```

Comment:
I forgot `this->`. It's not required in MSVC. It might be required in clang but I think not because PrivateHeap is declared in this class, not a superclass. I'll add it to clean it up if there are any other changes to be made from this review.